gradlew

```diff
@@ -205,6 +205,12 @@ set -- \
         org.gradle.wrapper.GradleWrapperMain \
         "$@"
 
+# Stop when "xargs" is not available.
+if ! command -v xargs >/dev/null 2>&1
+then
+    die "xargs is not available"
+fi
+
```

Comment:
When does this happen? I thought xargs comes by default with most unix systems.

Reply:
This file was generated when I ran command to upgrade gradle